src/main/cpp/DragonVision/DragonVision.h

```diff
@@ -19,8 +19,9 @@
 #include <string>
 
 #include "State.h"
-#include <DragonVision/DragonLimelight.h>
-#include <DragonVision/DragonVisionTarget.h>
+#include "DragonVision/DragonCamera.h"
+#include "DragonVision/DragonVisionTarget.h"
+#include "DragonVision/DragonVisonStructs.h"
```

Comment:
this is unnecessary and complicates maintenance

Reply:
removed excess includes